File: sdkwork-backend-sdk-kotlin/src/main/kotlin/com/sdkwork/backend/GenerationOutput.kt
package com.sdkwork.backend

data class GenerationOutput(
    val resources: List<AssetMediaResource>? = null,
    val primaryUrl: String? = null,
    val urls: List<String>? = null,
    val durationMs: Int? = null,
    val tokenUsage: Int? = null,
    val cost: Double? = null,
    val modelVersion: String? = null,
    val seed: Int? = null,
    val metadata: Map<String, Any>? = null,
    val createdAt: String? = null,
    val resourceCount: Int? = null,
    val primaryResource: AssetMediaResource? = null
)

File: sdkwork-backend-sdk-python/sdkwork_backend_sdk/models/generation_output.py
from dataclasses import dataclass
from typing import Optional, List, Dict, Any

@dataclass
class GenerationOutput:
    """AI生成任务输出结果"""
    resources: List[AssetMediaResource] = None
    primary_url: str = None
    urls: List[str] = None
    duration_ms: int = None
    token_usage: int = None
    cost: float = None
    model_version: str = None
    seed: int = None
    metadata: Dict[str, Any] = None
    created_at: str = None
    resource_count: int = None
    primary_resource: AssetMediaResource = None
